webapp/src/app/admin/teams/table/teams-table.component.html

```diff
@@ -0,0 +1,174 @@
+<div class="flex flex-col justify-between gap-4 sm:flex-row">
+  <input hlmInput class="w-full md:w-80" placeholder="Filter names..." [ngModel]="_nameFilter()" (ngModelChange)="_rawFilterInput.set($event)" />
+
+  <div class="flex items-center gap-4">
+    <button hlmBtn variant="outline" size="icon" (click)="invalidateTeams()">
+      @if (this.isLoading()) {
+        <hlm-icon name="lucideRotateCw" strokeWidth="2px" size="sm" class="animate-spin" />
+      } @else {
+        <hlm-icon name="lucideRotateCw" strokeWidth="2px" size="sm" />
+      }
+    </button>
+  </div>
+</div>
+
+<brn-table
+  hlm
+  stickyHeader
+  class="border-border mt-4 block overflow-auto rounded-md border"
+  [dataSource]="_filteredSortedPaginatedTeams()"
+  [displayedColumns]="_allDisplayedColumns"
+  [trackBy]="_trackBy"
+>
+  <brn-column-def name="name" class="w-32 sm:w-40 sm:ml-2">
+    <hlm-th *brnHeaderDef>
+      <button hlmBtn size="sm" variant="ghost" (click)="handleNameSortChange()">
+        Name
+        <hlm-icon class="ml-3" size="sm" name="lucideArrowUpDown" />
+      </button>
+    </hlm-th>
+    <hlm-td truncate *brnCellDef="let element">
+      @if (this.isLoading()) {
+        <hlm-skeleton class="h-6 w-20" />
+      } @else {
+        {{ element.name }}
+      }
+    </hlm-td>
+  </brn-column-def>
+  <brn-column-def name="color" class="w-28">
+    <hlm-th *brnHeaderDef>Color</hlm-th>
+    <hlm-td class="font-medium tabular-nums flex gap-2" *brnCellDef="let element">
+      @if (this.isLoading()) {
+        <hlm-skeleton class="w-4 h-4 rounded-full" [style.backgroundColor]="'#69feff'" />
+        <hlm-skeleton class="h-6 w-12" />
+      } @else {
+        <div class="w-4 h-4 rounded-full" [style.backgroundColor]="element.color"></div>
+        <span>{{ element.color }}</span>
+      }
+    </hlm-td>
+  </brn-column-def>
+  <brn-column-def name="repositories" class="w-48 flex-1">
+    <hlm-th *brnHeaderDef>Repositories</hlm-th>
```

Comment:
Repo name and repositories are not connected for me. Maybe for our repos that should already be be. done.
<img width="1141" alt="Bildschirmfoto 2024-11-11 um 12 11 34" src="https://github.com/user-attachments/assets/41b4fc8b-7cf1-4cd5-bd87-b0cbc35266c2">